smsdk/config/api_endpoints.json

```diff
@@ -4,26 +4,23 @@
   },
   "Cycle": {
     "url_v1": "/v1/datatab/cycle",
-    "url": "/api/cycle",
     "alt_url": "/api/cycle"
   },
   "Factory": {
     "url" : "/api/factory"
```

Comment:
This probably needs to be a separate ticket.  Maybe related to https://sightmachine.atlassian.net/browse/ENG-3228